crates/core/src/table/builder.rs

```diff
@@ -54,20 +30,13 @@ pub enum DeltaVersion {
 #[derive(Debug, Serialize, Deserialize, Clone)]
 #[serde(rename_all = "camelCase")]
 pub struct DeltaTableConfig {
-    /// Indicates whether our use case requires tracking tombstones.
-    /// This defaults to `true`
-    ///
-    /// Read-only applications never require tombstones. Tombstones
-    /// are only required when writing checkpoints, so even many writers
-    /// may want to skip them.
-    pub require_tombstones: bool,
```

Comment:
we never reference this config ... reading tomstones has been lazy for a while now.